### PR TITLE
FXIOS-3876 v101 String freeze

### PR DESCRIPTION
**Pull request by lmarceau**
# [FXIOS-3876](https://mozilla-hub.atlassian.net/browse/FXIOS-3876)
New strings for v101 version. 

### Search groups
```
FirefoxHomepage.Common.SitesCount.v101 = "%d sites"
```

### FirefoxHomeContextMenuHelper
```
HomePanel.ContextMenu.OpenInNewPrivateTab.v101 = "Open in a Private Tab"
```

### Homepage settings
```
Settings.Home.Option.Title.v101 = "Include on Homepage"
Settings.Home.Current.Description.v101 = "Choose what displays as the homepage."
Settings.Home.Option.Description.v101 = "Choose what’s included on the Firefox homepage."
```

### Inactive tabs
```
Settings.Tabs.CustomizeTabsSection.InactiveTabsDescription.v101 = "Tabs you haven’t viewed for two weeks get moved to the inactive section."
```

### CODE REVIEW

File: templates/firefox-ios.xliff [hunk 1]
```diff
@@ -1084,6 +1084,18 @@
         <source>Remove</source>
         <note>When a user taps and holds on an item from the Recently Visited section, this label will appear indicating the option to remove that item.</note>
       </trans-unit>
+      <trans-unit id="FirefoxHomepage.Common.SitesCount.v101" xml:space="preserve">
+        <source>%d sites</source>
+        <note>Label showing how many sites there is in a search group. %d represents a number</note>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:1089]: Unfortunately we don't support proper plurals at the time :/
To work around this, could you rephrase to `Sites: %d`
More info about this [here](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#use-proper-plural-forms), at the bottom of the section.

**Review by Delphine — CHANGES_REQUESTED** [review]
Just one quick fix and this will be good to go!

File: templates/firefox-ios.xliff (hunk 1 above)
Comment (lmarceau) [reply templates/firefox-ios.xliff:1089]: cc: @dnarcese just so you know, string will be "Sites: %d" instead

**Review by Delphine — APPROVED** [review]
LGTM now! thanks
